component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java

```diff
@@ -70,12 +72,13 @@ public static RadiusResponse sendRequest(String uid, String pass, byte[] state)
                 socket.send(packet);
                 socket.setSoTimeout(2500);
                 socket.receive(responsePacket);
-                return RadiusPacket.parsePacket(responsePacket);
+                return radiusPacket.parsePacket(responsePacket);
             } catch (IOException ioe) {
-                log.error("Could not get the Datagram responsePacket",ioe);
+                log.error("Could not get the Datagram responsePacket", ioe);
```

Comment:
Better to have whitespace when you print an error message, so can identify the error message and exception clearly.
   log.error("Could not get the Datagram responsePacket ", ioe);

Reply:
fixed